features/api/3ds_authorization.feature

```diff
@@ -31,3 +31,23 @@ Feature: A payment authorization request is made against the Worldpay payment ga
     And the response should reference the "42796904" order code
     And the response should reference a valid "paRequest" value
     And the response should reference the following issuerURL: "https://secure-test.worldpay.com/jsp/test/shopper/ThreeDResponseSimulator.jsp?orderCode=42796904"
+
+  Scenario: 3D Secure payment completion
+    When I complete the following payment
```

Comment:
The authorization for the payment is completed, instead of the payment itself.
Would `When the authorization for the following payment is completed` read better?